src/routes/confidentiality.route.js

```diff
@@ -4,7 +4,7 @@ const Constants = require('../constants')
 const {CONFIDENTIALITY, TASK_LIST} = Constants.Routes
 const ConfidentialityController = require('../controllers/confidentiality.controller')
 const ConfidentialityValidator = require('../validators/confidentiality.validator')
-const controller = new ConfidentialityController(CONFIDENTIALITY, true, TASK_LIST, ConfidentialityValidator)
+const controller = new ConfidentialityController(CONFIDENTIALITY, true, TASK_LIST, new ConfidentialityValidator())
```

Comment:
Something to discuss and possibly refactor later, but I think how you have passed in an instance of the validator here is probably how we should be setting up every route.

Reply:
To be refactored later.